README.md

```diff
@@ -2,32 +2,68 @@ Upstream status (`master` branch):
 [![OfflineIMAP build status on Travis-CI.org](https://travis-ci.org/OfflineIMAP/offlineimap.svg?branch=master)](https://travis-ci.org/OfflineIMAP/offlineimap)
 [![OfflineIMAP code coverage on Codecov.io](https://codecov.io/gh/OfflineIMAP/offlineimap/branch/master/graph/badge.svg)](https://codecov.io/gh/OfflineIMAP/offlineimap)
 [![Gitter chat](https://badges.gitter.im/OfflineIMAP/offlineimap.png)](https://gitter.im/OfflineIMAP/offlineimap)
+[![Say thanks to OfflineIMAP](https://img.shields.io/badge/Say%20Thanks-!-______.svg)](https://saythanks.io/to/OfflineIMAP)
 
 Upstream status (`next` branch):
 [![OfflineIMAP build status on Travis-CI.org](https://travis-ci.org/OfflineIMAP/offlineimap.svg?branch=next)](https://travis-ci.org/OfflineIMAP/offlineimap)
 
-[offlineimap]: http://github.com/OfflineIMAP/offlineimap
-[website]: http://www.offlineimap.org
-[wiki]: http://github.com/OfflineIMAP/offlineimap/wiki
-[blog]: http://www.offlineimap.org/posts.html
+[![Github All Releases](https://img.shields.io/github/downloads/atom/atom/total.svg)](https://github.com/OfflineIMAP/offlineimap/graphs/traffic)
```

Comment:
"All releases" pointing to the traffic? Please, remove this line.

Reply:
The "All releases" is just the image alternate text. The badge is for proudly displaying "total downloads" from github, which is over 1 Million!  The image alternate can be changed, to "Total downloads" or something like this, alright?

Reply:
I didn't see the "over 1 million" stat on the target page.

Reply:
It's true. The badge gets the 1 Million downloads data by posting a http get to the GitHub Traffic REST API. https://developer.github.com/v3/repos/traffic/   Linking to the project "traffic" page to see some similar data, is an interesting page to show the user, because there isn't really any page available to show the raw REST API statistics, and anyway, those raw data not presented in a nice html page, like the traffic statistics page is.

Reply:
What about moving this to the website (for the same reason as above)?